app/controllers/runners_controller.rb

```diff
@@ -35,11 +35,44 @@ def create
         format.json { render json: @runner.errors, status: :unprocessable_entity }
       end
     end
+    #Assigning all existing plans to runner as individual training plans
+    Grouptrainingplan.all.each do |plan|
```

Comment:
Same issue as in destory, can't you do a Grouptrainingplan.find(:groupid ==something)?